pandas/tests/resample/test_resample_api.py

```diff
@@ -611,3 +611,146 @@ def test_resample_agg_readonly():
 
     result = rs.agg("min")
     tm.assert_series_equal(result, expected)
+
+
+def test_backward_resample():
+    # GH#37804
+
+    start, end = "2000-10-01 23:30:00", "2000-10-02 00:26:00"
+    rng = date_range(start, end, freq="7min")
+    ts = Series(np.arange(len(rng)) * 3, index=rng)
+
+    # test consistency of backward and origin
```

Comment:
Can you split up this large test into multiple smaller tests? (one per `# comment` would be good)

Reply:
done, cc @mroeschke